api/samples/memval_simple.c

```diff
@@ -403,7 +421,7 @@ event_exit(void)
 DR_EXPORT void
 dr_client_main(client_id_t id, int argc, const char *argv[])
 {
-    drreg_options_t ops = {sizeof(ops), 3 /*max slots needed*/, false};
+    drreg_options_t ops = {sizeof(ops), 3 /*max slots needed*/, true};
```

Comment:
That being said, I can try using the default non-conservative approach and hope for the best, since logically `app_reg` should never be dead--we only call `drreg_get_app_value` when `where` depends on a register that `drreg` has reserved.